client/src/components/RegisterAccountForm/RegisterAccountForm.spec.js

```diff
@@ -146,14 +143,14 @@ describe('RegisterAccountForm', () => {
     expect(inputElement).toBeRequired()
   })
 
-  it('has 1 button element', () => {
+  it('has 2 button elements', () => {
     render(
       <Router>
         <RegisterAccountForm />
       </Router>
     )
 
     const buttonElements = screen.getAllByRole('button')
-    expect(buttonElements).toHaveLength(1)
+    expect(buttonElements).toHaveLength(2)
```

Comment:
It might make sense to separate the RegisterAccountForm component into two components. One for the form and one parent for the route and the axios request. This would then also allow you to write a more sophisticated test, that could check if a submit function is called when clicking the submit button, and not when clicking the back button.

Reply:
Ok, I'll add that to the todo list!